department-of-veterans-affairs/caseflow
# Craig/appeals 10142

Resolves [APPEALS-10142](https://vajira.max.gov/browse/APPEALS-10142)

### Description
- Added required veterans to the static_test_data.rb seed
- Added tests to spec file

### Acceptance Criteria
- [ ] Each of the veterans (list can be found in Jira) can be searched for in Caseflow and the data on case details matches the spreadsheet

### Testing Plan
1. Go to caseflowdemo environment
2. Go to search
3. Search for each veteran by SSN (the file numbers are set up to be their SSN)
4. Verify each veteran's data on case details page matches the spreadsheet data
